docs/API.md

```diff
@@ -728,11 +729,11 @@ An object with the following properties:
 
 Enqueues a new *test*.
 
-If an identical *test* was already enqueued and hasn't been started or was enqueued less than 10 minutes earlier,
-no new *test* is enqueued.
+If an identical *test* was already enqueued and hasn't been started or was enqueued
+less than [`age_reuse_previous_test`] seconds earlier, no new *test* is enqueued.
 Instead the id for the already enqueued or run test is returned.
```

Comment:
On lines 801ff there is a more user friendly text that says the same thing. Why not just refer to that?